server/gubled.go

```diff
@@ -212,10 +215,11 @@ func StartService() *service.Service {
 	srv.RegisterModules(4, 3, CreateModules(r)...)
 
 	if err = srv.Start(); err != nil {
+		logger.WithField("error", err.Error()).Error("errors occurred while starting service")
 		if err = srv.Stop(); err != nil {
-			logger.WithError(err).Error("Error when stopping service after Start() failed")
+			logger.WithField("error", err.Error()).Error("errors occured when stopping service after it failed to start")
 		}
-		logger.WithError(err).Fatal("Service could not be started")
+		return nil
```

Comment:
see the new code at line 170
here, I just wanted to move the "Fatal" to the upper/invocation level => so that the assertions in tests regarding the result of StartService actually mean something :)